packages/opencensus-resource-util/src/resource-labels.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Constants for collecting resource information. */
+// tslint:disable-next-line:variable-name
```

Comment:
Why not name this `AwsResourceKeyConstants` following the variable naming guide?

That rule is ultimately based on the Google JS style guide I believe: https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined

Reply:
Agreed, I wanted to group labels together based on types (GCP, AWS, K8s). Removed namespace https://github.com/census-instrumentation/opencensus-node/pull/299/commits/d994ca9cc34e554b24585c7eed7ab79ef184466a